test/src/examples_complete_test.go

```diff
@@ -0,0 +1,90 @@
+package test
+
+import (
+	"strings"
```

Comment:
Use regex in DisabledTest for cleaner output
```suggestion
	"regexp"
	"strings"
```